html/siptest.js

```diff
@@ -313,7 +313,11 @@ $(document).ready(function() {
 										}
 										$('.dtmf').click(function() {
 											// Send DTMF tone
-											sipcall.dtmf({dtmf: { tones: $(this).text()}});
+											// Inband
+											// sipcall.dtmf({dtmf: { tones: $(this).text()}});
```

Comment:
Please don't modify the demo, RFC2833 is a standard, SIP INFO DTMF is not (actually, some IETF meetings ago there were discussions about INFO being actually evil! :wink:). Besides, the main purpose of that demo is to also showcase how you can generate DTMF tones via WebRTC. Since the new backend would be there, I'd add a comment after where we currently send DTMF tones (e.g., something like "You can also send DTMF tones via SIP INFO this way" followed by the example) so that users are aware of the functionality.

I'd rather not send them both at the same time, s there are systems that would process them both thus leading to duplicated input and broken applications.